src/app/DeviceProxy.h

```diff
@@ -59,6 +59,25 @@ class DLL_EXPORT DeviceProxy
 
     const ReliableMessageProtocolConfig & GetRemoteMRPConfig() const { return mRemoteMRPConfig; }
 
+    /**
+     * @brief
+     *   This function returns the attestation challenge for the secure session.
+     *
+     * @param[out] attestationChallenge The output for the attestationChallenge
+     *
+     * @return CHIP_ERROR               CHIP_NO_ERROR on success, or CHIP_ERROR_INVALID_ARGUMENT if no secure session is active
+     */
+    virtual CHIP_ERROR GetAttestationChallenge(ByteSpan & attestationChallenge)
+    {
+        Optional<SessionHandle> secureSessionHandle;
```

Comment:
Might be better to put the implementation in the .cpp...

Reply:
Seems no updates in a while on this PR. Assuming this can be done as a followup.

Reply:
@andy31415 Is there an issue tracking tahg followup assigned to this PR's author?

I am fine with followups, but I want them to actually happen, and except for a few people our track record at them is very poor.

Reply:
Followup is here: https://github.com/project-chip/connectedhomeip/pull/19058